docs/concepts/11-fees.mdx

```diff
@@ -80,7 +80,7 @@ Airdrop campaigns created with earlier versions do not incur any claim fee.
 
 :::tip[What is the fee?]
 
-The fee is **$1** (in the gas token) per claim, regardless of the airdrop amount.
+The fee is ~**$1** (in the gas token) per claim, regardless of the airdrop amount.
```

Comment:
Why have an approximation here but not in the UI Fees?

Reply:
Because the UI will soon have oracles, while the contracts will use fixed fees for a while, so the actual dollar values may be different.